sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/CustomBearerTokenAuthenticationPolicy.java

```diff
@@ -23,6 +23,7 @@
 public class CustomBearerTokenAuthenticationPolicy implements HttpPipelinePolicy {
     private static final ClientLogger LOGGER = new ClientLogger(CustomBearerTokenAuthenticationPolicy.class);
     private static final String BEARER = "Bearer";
+    private static final HttpHeaderName X_FORWARDED_HOST = HttpHeaderName.fromString("X-FORWARDED-HOST");
```

Comment:
unrelated to this PR, but, wow, this should be populated by proxies and not SDKs.